### PR TITLE
return to edit mode on file upload actions

### PR DESCRIPTION
**Pull request by francodacosta**
file upload actions are done as part of the save action, to ensure any changes to form are saved if a file action is performed

click the save button redirects to list view, performing file actions keeps user in edit view

### CODE REVIEW

File: web/domains/case/access/views.py [hunk 1]
```diff
@@ -220,24 +220,49 @@ def delete(self, request, process_id):
...
+            )
+
+        form.save()
+
+        if 'uploaded_file' in request.FILES:
+            self.upload(request, process_id)
+            return self.edit(request, process_id)
+
+        if 'delete_file_id' in request.POST:
+            logger.debug('going to mark file %s as deleted' % request.POST['delete_file_id'])
```
Comment (rossmiller) [inline web/domains/case/access/views.py]: A matter of taste to some degree but I would use f-strings for formatting these days.

File: web/domains/case/access/views.py [hunk 2]
```diff
@@ -220,24 +220,49 @@ def delete(self, request, process_id):
 
         return redirect('access_request_fir_list', process_id=process_id)
 
+
+    @csrf_exempt
```
Comment (rossmiller) [inline web/domains/case/access/views.py]: Is this a security risk in any way?

Comment (francodacosta) [reply web/domains/case/access/views.py]: a bit, but s3 chunk uploader documentation explicitly says to disable csrf

Comment (rossmiller) [reply web/domains/case/access/views.py]: We agreed to add a spike to investigate why they made this recommendation and if it's wholesome.

**Pull request merged**